types/opentype.js/index.d.ts

```diff
@@ -1,65 +1,153 @@
-// Type definitions for opentype.js
+// Type definitions for opentype.js 0.7.3
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Reply:
@DanielRosenwasser this is now fixed